x/protocolpool/abci.go

```diff
@@ -0,0 +1,41 @@
+package protocolpool
+
+import (
+	"time"
+
+	"cosmossdk.io/x/protocolpool/keeper"
+	"cosmossdk.io/x/protocolpool/types"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
```

Comment:
```suggestion
func EndBlocker(ctx context.Context, k keeper.Keeper) error {
```
since the logger is on the keeper already we can use that. This way we get rid of unrwrapping in module.go and potentially remove sdk import here